packages/plans-grid-next/src/storybook-mocks/index.tsx

```diff
@@ -1,13 +1,9 @@
 import { getFeaturesList } from '@automattic/calypso-products';
-import { gridPlans } from './gird-plans';
-import { gridPlanForSpotlight } from './grid-plan-for-spotlight';
 
 export const defaultArgs = {
-	gridPlans,
-	gridPlanForSpotlight,
```

Comment:
Removing `gridPlanForSpotlight` as a default as it's not relevant for all configs eg. `/setup/newsletter`